spec/spec_helper.rb

```diff
@@ -10,7 +10,7 @@
     c.syntax = :expect
   end
 
-  config.before(:each) do
+  config.before(:all) do
```

Comment:
When this was set to `:each`, running the `transmitter_spec` tests would cause failures as they connection had not yet been properly configured at the time that the `TestHandler` was instantiated.